d3-progress-meter.html

```diff
@@ -186,12 +204,15 @@
 						.innerRadius(0.825 * this.radius)
 						.outerRadius(0.85 * this.radius);
 
-			this.$.current.style.fontSize = this.radius/2 + 'px';
-			this.$.goal.style.fontSize = this.radius/5 + 'px';
-			this.$.type.style.fontSize = this.radius/4 + 'px';
-			this.$.type.style.width = 2 * this.radius + 'px';
-			this.$.caption.style.fontSize = this.radius/5 + 'px';
-			this.$.caption.style.width = 2 * this.radius + 'px';
+			if(this.customStyle) {
```

Comment:
Can you please follow style of the component:
- Use tabs instead of spaces for indention
- Add space between `if(` -> `if (`